packages/next-server/server/next-server.ts

```diff
@@ -249,7 +250,7 @@ export default class Server {
     return this.sendHTML(req, res, html)
   }
 
-  public async renderErrorToHTML (err: Error|null, req: IncomingMessage, res: ServerResponse, pathname: string, query: ParsedUrlQuery = {}) {
+  public async renderErrorToHTML (err: Error|null, req: IncomingMessage, res: ServerResponse, _pathname: string, query: ParsedUrlQuery = {}) {
```

Comment:
Why is this param being renamed?

Reply:
It's unused, but there for backwards compat because it's a public API. When using underscore linters / vscode will ignore it as "you know what you're doing"